english/alchemy/alchemy.md

```diff
@@ -15,11 +15,10 @@ You decide to play it safe and repeat the process a few times. Who knows how man
 the high-score currently is or how many you will need on your adventure?
 
 The creation process gets quicker and quicker each time you practice the ancient art of
-alchemy. Your skills improve. However, on the fifth run, the coin has an air anomaly
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.